packages/shared/src/components/cards/v1/WelcomePostCard.tsx

```diff
@@ -1,18 +1,19 @@
-import React, { forwardRef, ReactElement, Ref, useRef } from 'react';
+import React, { forwardRef, ReactElement, Ref, useMemo, useRef } from 'react';
```

Comment:
Just realised we've been touching welcome post card unneccissarily, this is not needed for the feed layout changes. Let's revert the changes here and in useTruncatedSummary made specifically for this use case 🙏

Reply:
Damn, great point 😅 I completely forgot that these changes will not be applied to squad feeds... 🤦🏻‍♂️ 

Will revert, but I think we can keep the welcome card around even if not used

Reply:
we actually use the welcome card for freeform post types too: https://github.com/dailydotdev/apps/blob/1fbc96daf98d806b557ab63f55f37af3a9361674/packages/shared/src/components/FeedItemComponent.tsx#L105 -- so maybe we should keep it around, since these post types may show up in non-squad feeds

Reply:
Ah ok if so yes, all should be good